db/active_replicator_checkpointer.go

```diff
@@ -59,17 +59,19 @@ type Checkpointer struct {
 
 	// closeWg waits for the time-based checkpointer goroutine to finish.
 	closeWg sync.WaitGroup
+	// For logging stats about the length of expected and processed sequence lists
+	dbStats *base.DbReplicatorStats
```

Comment:
```suggestion
```

This isn't needed because we can directly access `c.stats.ExpectedSequenceLen`, etc. to update/read.

Reply:
https://github.com/couchbase/sync_gateway/pull/6098/commits/2eb80fbd8ae9af22fd8bb4eaf2e8a02c404034bf